src/site/content/en/notifications/codelab-notifications-get-started/index.md

```diff
@@ -0,0 +1,299 @@
+---
+layout: codelab
+title: Get started with the Notifications API
+description: |
+  In this codelab, learn how to request user permission and send notifications.
+date: 2019-10-15
+# Add an updated date to your post if you edit in the future.
+# updated: 2019-06-27
+glitch: codelab-notifications-get-started
+# Specify which file the glitch should start on.
+glitchPath: public/index.js
+related_post: use-push-notifications-to-engage-users
+---
+
+In this codelab, you'll use basic features of the
+[Notifications API](https://developer.mozilla.org/en-US/docs/Web/API/Notifications_API) to:
+
+* Request permission to send notifications
+* Send notifications
+* Experiment with notification options 
+
+## Remix the sample app and view it in a new tab
+
+Notifications are automatically blocked from the embedded Glitch app, so you won't be able to preview the app on this page. Instead, here's what to do:
+
+{% Instruction 'remix', 'ol' %}
+{% Instruction 'preview', 'ol' %}
+
+The Glitch should open in a new Chrome tab:
+
+<img class="w-screenshot" src="./live-app-new-tab.png" alt="Screenshot showing the remixed live app in a new tab">
+
+As you work through this codelab, make changes to the code in the embedded Glitch on this page. Refresh the new tab with your live app to see the changes.
+
+## Get familiar with the starting app and its code
+
+Start by checking out the live app in the new Chrome tab:
+
+1.  {% Instruction 'devtools-console', 'none' %}
+
+    You should see the following message in the Console:
+
+    ```text
```

Comment:
I switched from a screenshot to a code block to improve accessibility. Otherwise, assistive tech users won't know what the message is.